frontend/src/components/layout/Navbar.tsx

```diff
@@ -50,10 +50,8 @@ export const Navbar = () => {
             } else if (type === IdType.ACCOUNT) {
               navigate(`/account/${searchQuery}`);
             } else if (type === IdType.DOMAIN) {
-              const accId = await getAccountIdFromDomain(searchQuery);
-              if (accId) {
-                navigate(`/account/${accId}`);
-              }
+              const accId = '0.0.1005415';
+              navigate(`/account/${accId}`);
```

Comment:
We should undo the changes in this file since it breaks searching by domain and will result in all domains going to the dev account.

Other users will still be continuing to use domains :)